src/main/java/com/ft/methodearticlemapper/transformation/ImageSetXmlEventHandler.java

```diff
@@ -60,6 +61,22 @@ private String getUuidFromContext(final BodyProcessingContext bodyProcessingCont
         return uuid;
     }
 
+    private String getApiHostFromContext(final BodyProcessingContext bodyProcessingContext) {
```

Comment:
this is a copy/paste, please make a template the method and use that for both concrete cases.